<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ExportAttachClientPage1</class>
 <widget class="QDialog" name="ExportAttachClientPage1">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>368</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="7" column="0" colspan="4">
    <widget class="CProgressBar" name="progressBar">
     <property name="value">
      <number>24</number>
     </property>
    </widget>
   </item>
   <item row="8" column="0" colspan="4">
    <widget class="QTextBrowser" name="logBrowser"/>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="lblDate">
     <property name="text">
      <string>Дата</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="lblInsurer">
     <property name="text">
      <string>Плательщик</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="CDateEdit" name="edtDate">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="calendarPopup">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="lblOrgStructure">
     <property name="text">
      <string>Подразделение</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1" colspan="3">
    <widget class="CInsurerComboBox" name="cmbInsurer"/>
   </item>
   <item row="9" column="3">
    <widget class="QPushButton" name="btnCancel">
     <property name="text">
      <string>прервать</string>
     </property>
    </widget>
   </item>
   <item row="9" column="2">
    <widget class="QPushButton" name="btnExport">
     <property name="text">
      <string>экспорт</string>
     </property>
    </widget>
   </item>
   <item row="5" column="1" colspan="3">
    <widget class="CRBListBox" name="lstOrgStructure"/>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CProgressBar</class>
   <extends>QProgressBar</extends>
   <header>library/ProgressBar.h</header>
  </customwidget>
  <customwidget>
   <class>CDateEdit</class>
   <extends>QDateEdit</extends>
   <header>library/DateEdit.h</header>
  </customwidget>
  <customwidget>
   <class>CRBListBox</class>
   <extends>QListView</extends>
   <header>library/RBListBox.h</header>
  </customwidget>
  <customwidget>
   <class>CInsurerComboBox</class>
   <extends>QComboBox</extends>
   <header>Orgs/OrgComboBox.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
